### PR TITLE
I want to access a scratch org created by MetaDeploy.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

- https://trello.com/c/1u5fgIH3

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x] Description is in Trello card

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- After successful installation on scratch org, click "View Scratch Org" link at bottom of success modal.

## Show me
_Provide screenshots/animated gifs/videos if necessary._

Screen Shot 2020-11-06 at 3 58 08 PM

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
@davisagli This is largely copied over from Metecho, and seems to work well functionally. @jerivas is working on triggering the pwd-reset on the org, but in the meantime I'm happy for your code review here (also yours @jerivas) at your leisure. 👍

**Comment by jgerigmeyer** [comment]
[I want to access a scratch org created by MetaDeploy.](https://trello.com/c/1u5fgIH3/158-i-want-to-access-a-scratch-org-created-by-metadeploy)

File: metadeploy/api/models.py [hunk 1]
```diff
@@ -939,6 +940,23 @@ def complete(self, config):
         self.save()
         async_to_sync(notify_org_finished)(self)
 
+    def get_refreshed_org_config(self):
+        org_config = OrgConfig(self.config, self.plan.org_config_name)
+        info = jwt_session(
+            settings.CONNECTED_APP_CLIENT_ID,
+            settings.CONNECTED_APP_CLIENT_KEY,
```
Comment (davisagli) [inline metadeploy/api/models.py:947]: FWIW, if these settings happened to come from environment variables names SFDX_CLIENT_ID and SFDX_HUB_KEY, then all we would need to do is `org_config.refresh_oauth_token(None)` and CumulusCI would do the rest of this based on the presence of those environment variables. (Without them, that method uses a different approach for refreshing the token which won't work here.)

**Review by davisagli — COMMENTED** [review]
LGTM

File: metadeploy/api/models.py (hunk 1 above)
Comment (jerivas) [reply metadeploy/api/models.py:964]: @davisagli do you think that's a worthwhile change? These env vars are only used in a couple places in the code, env files, and CI config. I can take a shot at it on a separate PR.

**Review by jerivas — APPROVED** [review]

File: metadeploy/api/models.py (hunk 1 above)
Comment (davisagli) [reply metadeploy/api/models.py:964]: @jerivas It's pretty low priority. You could add it to a list of cleanup tasks to do if you have time while waiting for UX / frontend at some point.

Comment (jerivas) [reply metadeploy/api/models.py:964]: I've added it to the ice box

**Pull request merged**